src/main/java/com/softserve/entity/Room.java

```diff
@@ -44,4 +44,6 @@
     @Column(name = "disable",  columnDefinition = "boolean default 'false'")
     private boolean disable = false;
 
+    @Column(name = "sort_order")
+    private Double sortOrder;
```

Comment:
At the moment, the implementation of the SortOrder next: We choose after which room we want to insert our, then check the SortOrder of the next and insert between them on the principle: (AfterRoom + BeforeRoom) / 2.
 Example: We want to insert Room245 after Room142 (SortOrder = 2) and before Room156(SortOrder = 3)  so it will be: (2+3)/2 = 2.5
So our rooms order will be next: Room1, Room142, Room245, Room156. Because in develop branch ordering makes by title and we want to our custom order.

Reply:
I don't think that this is correct :) 
If so then we can even put 100 rooms in order between 1 and 2 )
I think this should be re-designed